apps/member/src/pages/GroupPage/GroupPage.tsx

```diff
@@ -1,18 +1,64 @@
+import { useEffect, useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 
-import { Button } from '@clab-platforms/design-system';
+import { Button, Menubar } from '@clab-platforms/design-system';
 
 import Content from '@components/common/Content/Content';
+import EmptyBox from '@components/common/EmptyBox/EmptyBox';
 import Header from '@components/common/Header/Header';
 import Section from '@components/common/Section/Section';
 import GroupCard from '@components/group/GroupCard/GroupCard';
 
+import { MY_MESSAGE } from '@constants/message';
 import { PATH } from '@constants/path';
-import { useActivityGroupMember } from '@hooks/queries';
+import { ACTIVITY_MEMBER_STATE, ACTIVITY_STATE } from '@constants/state';
+import {
+  useActivityGroupMember,
+  useActivityGroupMemberApplied,
+  useMyActivityGroupMember,
+} from '@hooks/queries';
 
+import type { ActivityGroupItem } from '@type/activity';
+
+const MenubarItems = [
+  { name: '현재 진행중인 그룹', value: ACTIVITY_STATE.PROGRESSING },
+  { name: '나의 활동', value: ACTIVITY_MEMBER_STATE.ACCEPTED },
+  { name: '나의 지원', value: ACTIVITY_MEMBER_STATE.WAITING },
+];
+const SubMenubarItems = [
+  { name: '진행중', value: ACTIVITY_STATE.PROGRESSING },
+  { name: '종료', value: ACTIVITY_STATE.END },
+];
 const GroupPage = () => {
   const navigate = useNavigate();
-  const { data } = useActivityGroupMember();
+  const [menu, setMenu] = useState(MenubarItems[0]);
+  const [subMenu, setSubMenu] = useState(SubMenubarItems[0]);
+  const { data: progressingData } = useActivityGroupMember();
+  const { data: waitingData } = useActivityGroupMemberApplied();
+  const { refetch, data: acceptedData } = useMyActivityGroupMember({
+    status: subMenu.value,
+  });
```

Comment:
`data`들에 대해서 구분을 해 주고 있다면 `refetch`도 어떤 데이터와 관련된 것인지 명확하게 나타내보는건 어떨까요?

```suggestion
  const { data: progressingData } = useActivityGroupMember();
  const { data: waitingData } = useActivityGroupMemberApplied();
  const { refetch: acceptedDataRefetch, data: acceptedData } = useMyActivityGroupMember({
    status: subMenu.value,
  });
```